package.json

```diff
@@ -203,7 +204,8 @@
     "test-cypress-open": "./bin/build-for-test && yarn test-cypress-no-build --open",
     "test-cypress-open-no-backend": "E2E_HOST='http://localhost:3000' yarn test-cypress-no-build --open",
     "test-cypress-no-build": "yarn && CONFIG_FILE=frontend/test/__support__/e2e/cypress.json babel-node ./frontend/test/__runner__/run_cypress_tests.js",
-    "test-cypress-smoketest": "yarn test-cypress-no-build --folder frontend/test/metabase-smoketest"
+    "test-cypress-smoketest": "yarn test-cypress-no-build --folder frontend/test/metabase-smoketest",
+    "prepare": "husky install"
```

Comment:
Can this script name be a bit more specific, please?
"install-husky", or anything similar.

Reply:
@nemanjaglumac It is a special lifecycle npm script that husky relies on https://typicode.github.io/husky/#/?id=install

Reply:
A note for the future (when we hopefully upgrade `yarn`):

> Yarn 2 doesn't support prepare lifecycle script, so husky needs to be installed differently (this doesn't apply to Yarn 1 though). See [Yarn 2 install](https://typicode.github.io/husky/#/?id=yarn-2).